### DISCUSSION TITLE
Tracking escrow balance for mortgage accounts

### DISCUSSION THREAD

**Discussion by Stugg** [firefly-iii/firefly-iii#5091] [Firefly III]
I'm trying to find a way to accurately track the escrow balance for my mortgage. I have the mortgage account set up as a Liabilities account and the escrow account as an Asset account. My monthly mortgage payment is comprised of three categories; a principal payment, an interest payment and an escrow payment.

I tried to split the entire monthly payment within an Expense transaction, as that allowed me to associate the entire sum with a Bill. I can set the Destination Account to the Liability in one split (principal payment), and select a different account for the interest split, but I can't select the escrow Asset account for the remaining portion since that would be a transfer.

I've been splitting out the escrow portion to a separate Transfer transaction, but this doesn't allow me to associate it with a Bill.

Any ideas on how to track this so the monthly billing is accurate but still allow me to keep tabs on my escrow account balance?

**Comment by JC5** [comment]
I'm not really sure what the concept of the escrow is. Could you elaborate?

**Reply by Stugg** [reply]
Hi, James. Thanks for the quick response and for all your work on this amazing application.

To simplify the math, let's say the mortgage payment is $1,000 per month. $300 of that goes towards principal, $300 towards interest, and the remaining $400 is placed in an account that's held in escrow by my mortgage company. Annual payments are pulled from the escrow account to cover my home insurance premiums and property taxes.

I'd like to track the balance of the escrow account over the year, then enter expense transactions from the escrow asset account when the insurance and taxes are paid out.

**Reply by JC5** [reply]
No problem, glad to do it.

Would it fit if the escrow account is also an asset account? Even though you can't technically touch it?

**Reply by Stugg** [reply]
An asset account would be ideal, and is how I currently have it set up. That allows me to track the balance as it increases with each monthly payment and disburse payments for insurance and taxes at end of year.

What I'm struggling with is the monthly payments. I can split my monthly mortgage payment to send the principal portion to a liability account and the interest portion to an expense account, but it's not allowing me to split the escrow portion to an asset account within the same transaction as it sees that as a transfer.

When I split the escrow to a transfer, I can't associate it with a bill so it throws off the bills and budgeting portion.

**Reply by JC5** [reply]
Yes, this isn't possible at the moment, I'll think about a solution.

**Comment by octagons** [comment]
@JC5 I am in the exact same boat as @Stugg. Just wanted to chime in to say that there is at least one other user attempting to do this!

On its face, the easiest solution seems to be allowing an asset account to be the target of a withdrawl split - perhaps the asset account can have a special flag to allow this?

**Comment by NovaBiele** [comment]
@JC5 I'm going to chime in here as well as I'd love to be able to utilize this exact functionality as described by @Stugg and @octagons.

**Comment by dwildstr** [comment]
It's several years later, but just popping by to say I too would like this capability (or if it's already been implemented, it's not obvious how to access it).

**Comment by slalomnut** [comment]
Bump.  Add another to the wish list.

**Comment by Charli181** [comment]
another Bump. In Australia, it is called a redraw facility. Example -  your mortgage payment is $500 per month and any extra you pay will also help reduce the interest on the principal of the loan, however you call also pull that extra money you payed back out of the loan at later stage. I am finding it impossible to track the amount I have available to "Redraw". 

Something like a Piggy bank linked to a loan, where the piggy bank is a positive value, and this also applied to the loan balance as well. eg

loan account owed is -10,000
piggy bank is 5,000
loan balance would show as -5,000

**Comment by nigtreg** [comment]
***@***.*** desde HUAWEI MatePad T-------- Mensaje original --------De: Charli181 ***@***.***>Fecha: dom., 9 mar. 2025 17:36Para: firefly-iii/firefly-iii ***@***.***>Cc: Subscribed ***@***.***>Asunto: Re: [firefly-iii/firefly-iii] Tracking escrow balance for mortgage accounts (Discussion #5091)
another Bump. In Australia, it is called a redraw facility. Example -  your mortgage payment is $500 per month and any extra you pay will also help reduce the interest on the principal of the loan, however you call also pull that extra money you payed back out of the loan at later stage. I am finding it impossible to track the amount I have available to "Redraw".
Something like a Piggy bank linked to a loan, where the piggy bank is a positive value, and this also applied to the loan balance as well. eg
loan account owed is -10,000
piggy bank is 5,000
loan balance would show as -5,000
